test_folder/components/Test/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
test_folder. I thought we are generating this programmatically only when running unit tests.

Reply:
OK, I think the same.

Reply:
I create that with cli from terminal. Just to show.. I'll remove that. Not a problem.

Reply:
Just saved that accidentally when used cli manually.

Reply:
oh cool